src/lib/ops.ts

```diff
@@ -2,24 +2,45 @@ import { ChildProcessWithoutNullStreams, execSync, SpawnOptionsWithoutStdio } fr
 
 type Spawn = (command: string, args?: readonly string[] | undefined, options?: SpawnOptionsWithoutStdio | undefined) => ChildProcessWithoutNullStreams;
 
-export default class Ops {
+export interface BuildOptions {
+  imageName?: string;
+  mounts?: string;
+}
+
+export interface RunOptions {
+  configPath?: string;
+  imageName?: string;
+  mounts?: string;
+}
+
+export interface StartInstanceOptions {
+  instanceName?: string;
+  ports?: string;
+  udpPorts?: string;
+}
+
+export class Ops {
   spawn: Spawn;
 
   constructor(_spawner: Spawn) {
     this.spawn = _spawner;
   }
 
-  build(filePath: string, configPath: string): ChildProcessWithoutNullStreams {
+  build(filePath: string, options: BuildOptions): ChildProcessWithoutNullStreams {
     let args = ["build", filePath];
 
-    if (configPath) {
```

Comment:
Are we losing the ability to use a config file to build images?

Reply:
oh sorry, that code is currently not being used because build command is not asking config file. I will add it back

Reply:
added back by https://github.com/nanovms/ops-vscode/pull/13/commits/86b372bf80cabfdd7624c6f56a1bddf840f64a4e

Reply:
Probably we should create new issue to add "Build with Config" command ?

Reply:
You're right. It's a useful command.